docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -398,6 +398,43 @@ def downstream_asset(upstream):
 
 In this case, `ins={"upstream": AssetIn("upstream_asset")}` declares that the contents of the asset with the key `upstream_asset` will be provided to the function argument named `upstream`.
 
+### Attaching metadata
+
+Dagster supports attaching arbitrary metadata to assets. To attach arbitrary metadata, supply a `metadata` dictionary to the asset:
```

Comment:
Make "metadata" a link to the `MetadataEntry` [API docs](https://docs.dagster.io/_apidocs/ops#dagster.MetadataEntry). Also no need for "arbitrary" twice.